frontend/src/components/common/Scenarios/ScenarioPlot.js

```diff
@@ -44,7 +46,7 @@ const ScenarioPlot = ({ scenarioData, scenarioSelectedRuptures, extra }) => {
           type: "scatter",
           mode: "lines",
           line: { color: deafultColours[colourCounter % 10] },
-          name: `Scenario ${curRup} [16, 50, 84 %ile]`,
+          name: `Scenario ${curRup} [mean and 16<sup>th</sup>, 84<sup>th</sup> percentile]`,
```

Comment:
Imo these should also be renamed, since its no longer the 16th and 84th. Maybe something like $\mu \pm \sigma$?

Not really sure on this though haha

Reply:
Isn't /pm 1 /sigma the equivalent to the 16th/84th percentile?

Reply:
Are they no longer 16th/84th percentiles?